controller/work_item_children_blackbox_test.go

```diff
@@ -834,3 +834,35 @@ func (s *searchParentExistsSuite) TestSearchWorkItemListFilterUsingParentExists(
 	})
 
 }
+
+func (s *workItemChildSuite) TestCreateAndDeleteLinkToChildrenThenListChildren() {
+	// create workitem links then remove it
+	workitemLink1 := s.linkWorkItems(s.T(), "bug1", "bug2")
```

Comment:
Please use test fixture for new tests instead.

Reply:
handled here -> 9888d7b